test/UniswapV2PairTest.sol

```diff
@@ -145,6 +227,25 @@ contract UniswapV2PairTest is Test {
     function getStubToken(uint256 mintAmount_) private returns (IERC20) {
         return new StubERC20(mintAmount_);
     }
+
+    function divCeil(uint numerand, uint divisor) private returns (uint) {
+        uint roundUpAmount = numerand % divisor == 0 ? 0 : 1;
+        return (numerand / divisor) + roundUpAmount;
+    }
+
+    function min(uint a, uint b) private returns (uint) {
+        return a < b ? a : b;
+    }
+
+    function max(uint a, uint b) private returns (uint) {
+        return a > b ? a : b;
+    }
+
+    function mapSeedToRange(uint seed, uint minInclusive, uint maxInclusive) private returns (uint) {
```

Comment:
I thought all of the +1's to make parameters exclusive for the max were confusing the calling logic, so I just made both parameters inclusive.